src/types/products.ts

```diff
@@ -35,6 +35,16 @@ type LearnProductName = Exclude<
 	'HashiCorp Cloud Platform' | 'Sentinel'
 >
 
+type HcpProductName = Exclude<
+	ProductName,
+	'HashiCorp Cloud Platform' | 'Nomad' | 'Sentinel' | 'Terraform' | 'Vagrant'
+>
+
+type HcpProductSlug = Exclude<
+	ProductSlug,
+	'hcp' | 'nomad' | 'sentinel' | 'terraform' | 'vagrant'
+>
```

Comment:
Did we opt to use `Exclude` here instead of `Include` because the exclusion list will assumedly continue to get smaller as more products are added to HCP?

Reply:
Yep! Instead of adding we can just delete. I'm fine with either tho